src/main/java/user/createinternship/CreateInternshipResponseDS.java

```diff
@@ -0,0 +1,13 @@
+package user.createinternship;
+
+public class CreateInternshipResponseDS {
+    private final String success_status;
```

Comment:
use service.ServerStatus, so this would be `private final ServerStatus status`. Also potentially add a message to return as well? That would allow you to return the message that users should be notified about if they have errors